Dockerfile

```diff
@@ -25,6 +25,10 @@ RUN export YARN_CACHE_FOLDER="$(mktemp -d)" \
 COPY --from=builder /app/dist /action-release/dist/
 RUN chmod +x /action-release/dist/index.js
 
+# move the sentry-cli binary to where the entrypoint expects it
+RUN mv /action-release/node_modules/@sentry/cli/sentry-cli /action-release/sentry-cli
```

Comment:
How do you know the exact path?

Reply:
@armenzg that's just where it gets installed when we install the CLI package, I just found it